### ISSUE TITLE
Globally configure EqualsVerifier

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jqno/equalsverifier#72]
```
What feature do you request?

A way to globally configure EqualsVerifier.


What problem will this feature solve?

Sometimes, you want to use EqualsVerifier the same way on several classes. For 
example, you want to use the same PrefabValues and usingGetClass() on all of 
your EqualsVerifier tests. Currently, you have to manually add them every time.

It would be nice to have a way to do this only once.


Please provide any additional information below.

It should still be possible to deviate from this configuration, or not use it 
at all.

This issue is split off from Issue 71 (see comments 5 and 6).

```

Original issue reported on code.google.com by `jan.ouw...@gmail.com` on 6 Jan 2013 at 5:06

**Comment by jqno** [comment]
I'm going to close this issue, since I added it myself anyway.

This is one way of dealing with many EqualsVerifier tests in the same codebase, but it's not the only one. I'm not sure how people are handling that at the moment, and I'm no longer convinced a global configuration is the best way to go.
